Resources/string.csv

```diff
@@ -85,148 +85,148 @@ TeamImpostor,Team Impostors,インポスター陣営,内鬼阵营,狼人陣營
+WatcherInfo,Keep your eyes on everyone's votes,みんなの投票に目を光らせよう,窥探他人的投票,注意所有人的投票
 
 # 特殊クルー役職
-BaitInfo,Bait your enemies,敵を罠にはめよう,诱骗敌人，让敌人落入你的陷阱,"犧牲小我,完成大我"
-LighterInfo,Light all over,すべてを照らそう,照亮黑暗，让别人看清楚,照亮黑暗中的一切吧
-MayorInfo,Your vote counts twice,自分の票が何倍もの力を持っている,你的投票更有分量,"你的票數更多,更有料"
-SabotageMasterInfo,Fix sabotages faster,より早くサボタージュを直そう,请尽快修理破坏,盡快維修船上的破壞
-SheriffInfo,Shoot the Impostors,インポスターを撃ち抜け,把内鬼统统执法掉,執法狼人們
-SnitchInfo,Finish your tasks to find the Impostors,タスクを早く済ませよう,完成任务来找出内鬼,完成你的所有任務來找到狼人們
-SpeedBoosterInfo,Boost the players,走らせろ,让其他人跑起来吧,讓其他人跑得更快
+BaitInfo,Bait your enemies,敵を罠にはめよう,诱骗敌人,犧牲小我,完成大我
+LighterInfo,Light all over,すべてを照らそう,夜空中最亮的星,照亮黑暗中的一切吧
+MayorInfo,Your vote counts twice,自分の票が何倍もの力を持っている,你的投票更有分量,你的票數更多,更有料
+SabotageMasterInfo,Fix sabotages faster,より早くサボタージュを直そう,快速修理飞船,盡快維修船上的破壞
+SheriffInfo,Shoot the Impostors,インポスターを撃ち抜け,把内鬼毙了,執法狼人們
+SnitchInfo,Finish your tasks to find the Impostors,タスクを早く済ませよう,完成任务以找到内鬼,完成你的所有任務來找到狼人們
+SpeedBoosterInfo,Boost the players,走らせろ,让他人跑得更快,讓其他人跑得更快
 DoctorInfo,Access vitals and cause of death,斯くして奴は死んだ,调查他人死因,看見他們的死因
-TrapperInfo,Trap your enemies,敵を罠にはめよう,引诱敌人落入你的陷阱当中,讓你的敵人受困於陷阱中
-DictatorInfo,Hang your enemy on your own judgement,独裁政治をしよう,让我统治世界，拥有无上的权利,讓所有人臣服於你
+TrapperInfo,Trap your enemies,敵を罠にはめよう,困住敌人,讓你的敵人受困於陷阱中
+DictatorInfo,Hang your enemy on your own judgement,独裁政治をしよう,世界啊，臣服于我！,讓所有人臣服於你
 
 # 第三陣営役職
-ArsonistInfo,Let them burn,燃やせ,火焰给我燃烧起来吧！,"燒吧,燒吧,燃燒吧"
-JesterInfo,Get voted out,追放されよう,想办法让人怀疑把你投出去,想個辦法讓你被投出去
-OpportunistInfo,Stay alive,とにかく生き残りましょう,苟着吧，苟也是个不错的选择,做點什麼事來讓你存活
-SchrodingerCatInfo,Get killed and join killer team,殺された陣営に所属しろ,加入击杀你的阵营,打不過就加入
-CSchrodingerCatInfo,You are now on Team Crewmates,クルーメイトの味方になった,你加入了船员阵营，要帮助船员找出内鬼,"你現在變成船員了,幫助船員們找出狼和中立帶刀者"
-MSchrodingerCatInfo,You are now on Team Impostors,インポスターの味方になった,你加入了内鬼阵营，要掩护内鬼同伴,"你現在變成狼人了,但是你不能砍人,盡量掩護你的隊友的身分"
-EgoSchrodingerCatInfo,You are now on Team Egoists,エゴイストの味方になった,你成为了自私之人的伙伴,"你是利己主義者的跟班"
-JSchrodingerCatInfo,I became an ally of my Jackal,ジャッカルの味方になった,你是豺狼的跟班，传宗接代是个好选择,"你是豺狼的跟班,盡量幫助他"
-TerroristInfo,Die after finishing your tasks,タスクを済ませ、自爆しよう,都别想活着，和我一起同归于尽！,"一個一個來,都別想給我活!"
-ExecutionerInfo,Get your target voted out,ターゲットを追放させよう,不惜一切代价把你的目标票出去！,"想出一切辦法來投出你的目標"
-EgoistInfo,Replace the Impostor win,インポスター勝利を独占しよう,我们要把属于内鬼的胜利给夺走！,讓我們來奪取狼人的勝利
-LoversInfo,Survive with your lover,恋人と生きて幸せを掴もう,你坠入了爱河，成为了一对恋人，一起活到最后吧！,你墜入了愛河(心理)
-JackalInfo,Kill Everyone,すべてを殺せ,快去杀光所有人，一只苍蝇都不要剩下！,殺光所有人不留活口
+ArsonistInfo,Let them burn,燃やせ,用火焰净化一切,燒吧,燒吧,燃燒吧
+JesterInfo,Get voted out,追放されよう,想办法被投出去,想個辦法讓你被投出去
+OpportunistInfo,Stay alive,とにかく生き残りましょう,苟且偷生,做點什麼事來讓你存活
+SchrodingerCatInfo,Get killed and join killer team,殺された陣営に所属しろ,成为杀害你的玩家的仆从,打不過就加入
+CSchrodingerCatInfo,You are now on Team Crewmates,クルーメイトの味方になった,你是船员的仆从,你現在變成船員了,幫助船員們找出狼和中立帶刀者
+MSchrodingerCatInfo,You are now on Team Impostors,インポスターの味方になった,你是内鬼的仆从,你現在變成狼人了,但是你不能砍人,盡量掩護你的隊友的身分
```

Comment:
CSV must be enclosed in double quotation marks such as "xxxxxxxxx,yyyyyyy" when a comma is included in the text.